example/src/screens/calendarsList.js

```diff
@@ -1,17 +1,87 @@
 import React, {Component} from 'react';
-import {CalendarList} from 'react-native-calendars';
+import {View, Text} from 'react-native';
+import {CalendarList, LocaleConfig} from 'react-native-calendars';
+
+export const styles = {
+  containerHeader: {
+    justifyContent: 'space-between',
+    flexDirection: 'row',
+    paddingTop: 10,
+    paddingBottom: 10,
+    borderShadow: 'none'
+  },
+  textMonth: {
+    fontSize: 25,
+    justifyContent: 'center',
+    alignItems: 'center',
+    color: '#10004F'
+  },
+  textYear: {
+    color: '#5C5C5C',
+    fontSize: 20,
+    justifyContent: 'center',
+    alignItems: 'center',
+    top: 5
+  },
+  customWeekStyle: {
+    color: '#138D86'
+  }
+};
+
+// const defaultBaseText = {
+//   width: 48,
+//   height: 48,
+//   borderWidth: 1,
+//   borderColor: '#bbb',
+//   justifyContent: 'flex-end',
+//   alignItems: 'flex-end',
+//   borderRadius: 8,
+//   padding: 2,
+//   margin: 0
+// };
+
+const themeLatam = {
+  todayTextStyle: {
+    color: '#138D86'
+  },
+  selectedStyle: {
+    borderRadius: 8,
+    borderColor: 'rgba(45, 52, 206, 0.15)'
+  },
+  dayHeaderStyle: {
+    fontFamily: 'LatamSans-Bold',
```

Comment:
Please change the font to something that every one his in their computers.
When I loaded your PR I had an error that I am missing the font.